components/switch/switch-mixin.js

```diff
@@ -192,7 +186,6 @@ export const SwitchMixin = superclass => class extends FocusMixin(RtlMixin(Focus
 		const text = html`<span id="${this._textId}" class="d2l-switch-text">${this.text}</span>`;
 		const textPosition = (this.textPosition === 'start' || this.textPosition === 'hidden'
 			? this.textPosition : 'end');
-		const tooltip = (this.tooltip ? html`<d2l-tooltip for="${this._switchId}">${this.tooltip}</d2l-tooltip>` : '');
```

Comment:
Note that `this._switchId` is no longer needed with this removed.  It is assigned below on the `div`, and initialized in the `constructor`.  I don't _think_ it will be needed for the other stories.  If you do, it's fine to keep it around since those changes are coming shortly, otherwise I would remove it.

Reply:
Yup, I agree. Thanks for pointing that out!